### PR TITLE
fix(i18n): Update nl-NL.json

### PR DESCRIPTION
**Pull request by jan-vandenberg**
Refinements

### Additional context

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Translations update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide related snapshots or videos.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

**Comment by jan-vandenberg** [comment]
This one looks like it's based on an older version of the file: https://github.com/elk-zone/elk/pull/1214 (but are also good improvements).

File: locales/nl-NL.json [hunk 1]
```diff
@@ -239,24 +239,24 @@
         "subscription_error": {
           "clear_error": "Wis error",
           "permission_denied": "Geen toestemming: zet notificaties aan in je browser.",
-          "request_error": "Er is een error tijdens het ophalen van de subscriptie, probeer opnieuw en als de error blijft, raporteer het probleem naar de Elk repository.",
-          "title": "Kon niet subscriben aan push notificaties"
+          "request_error": "Er is een error tijdens het ophalen van het lidmaatschap, probeer opnieuw en als de error blijft, raporteer het probleem naar de Elk repository.",
+          "title": "Kon niet lid worden van de pushnotificaties"
```
Comment (haayman) [inline locales/nl-NL.json:243]: isn't 'abonnement' or 'abonneren' better?

**Comment by joranquinten** [comment]
Can we combine the two into one perhaps?

File: locales/nl-NL.json (hunk 1 above)
Comment (jan-vandenberg) [reply locales/nl-NL.json:243]: Agree. Abonnement for 'subscriptie' and 'abonneren' for 'lid worden'

**Comment by patak-dev** [comment]
@jan-vandenberg merged #1214 so it is easier to merge it into this one. Please merge main to the PR and resolve the conflicts and we can then merge this PR too.

File: locales/nl-NL.json [hunk 2]
```diff
@@ -99,6 +99,7 @@
     "account_not_found": "Account {0} niet gevonden",
     "explore-list-empty": "Er is nu niets trending. Kom later terug!",
     "file_size_cannot_exceed_n_mb": "Bestandsgrootte mag niet groter zijn dan {0}MB",
+    "sign_in_error": "Kan geen verbinding maken met de server.",
     "sign_in_error": "Kan geen connectie maken met de server.",
```
Comment (joranquinten) [inline locales/nl-NL.json:103]:
```suggestion
```

**Review by joranquinten — COMMENTED** [review]
Deduped a JSON key

**Review by patak-dev — APPROVED** [review]

**Pull request merged**